src/app/core/components/privacy-statement/privacy-statement.component.scss

```diff
@@ -14,6 +14,10 @@
     h1 {
       font-size: 52px;
       font-weight: 700;
+      @media only screen and (max-width: 480px) {
+        font-size: 32px;
+        font-weight: 700;
+      }     
```

Comment:
We should define a common way of dealing with page headings throughout all pages.
Similar topic to the different heading colors.

CC @PotSchmoell probably a discussion point for our weekly - I could prepare a proposal

Reply:
I agree - that's what I thought, too. Let's discuss this.